spectral_cube/utils.py

```diff
@@ -38,7 +38,12 @@ def wrapper(self, *args, **kwargs):
         accepts_how_keyword = 'how' in argspec.args or argspec.varkw == 'how'
 
         warn_how = accepts_how_keyword and ((kwargs.get('how') == 'cube') or 'how' not in kwargs)
-        
+
+        # This restores showing the "loading the entire cube into memory" warning for
+        # _apply_everywhere and _cube_on_cube_operation
+        if function.__name__ in ['_apply_everywhere', '_cube_on_cube_operation']:
```

Comment:
@keflavich -- can you check if this is consistent with the changes you made here: https://github.com/radio-astro-tools/spectral-cube/commit/cb8dfbe37a5ed06870d9d08b3da5dc366aa12c0d